apps/heureka/src/components/Services/common/ImageVersionDetailsPanel/ImageVersionDetailsPanel.tsx

```diff
@@ -4,108 +4,126 @@
  */
 
 import React, { useState } from "react"
-import { Panel, PanelBody, Stack, Badge, Pill, Label } from "@cloudoperators/juno-ui-components"
+import { Panel, PanelBody, Stack, Badge, Pill, Label, ContentHeading } from "@cloudoperators/juno-ui-components"
+import { MessagesProvider, Messages } from "@cloudoperators/juno-messages-provider"
 import { ServiceImageVersion } from "../../utils"
+import { ImageVersionIssuesList } from "./ImageVersionIssuesList"
 
 type ImageVersionDetailsPanelProps = {
   imageVersion: ServiceImageVersion
+  serviceCcrn: string
   onClose: () => void
 }
 
-export const ImageVersionDetailsPanel = ({ imageVersion, onClose }: ImageVersionDetailsPanelProps) => {
+export const ImageVersionDetailsPanel = ({ imageVersion, serviceCcrn, onClose }: ImageVersionDetailsPanelProps) => {
   const [showOccurrences, setShowOccurrences] = useState(false)
 
   return (
-    <Panel heading={`Image ${imageVersion.repository} Information`} opened={true} onClose={onClose} size="large">
-      <PanelBody>
-        <Stack gap="6" direction="vertical" className="w-full">
-          <Stack gap="4" direction="vertical">
-            {/* Component Details Row */}
-            <Stack gap="2" direction="horizontal">
-              <Label text="Image Details: " />
-              <Stack direction="horizontal" gap="2" wrap>
-                <Pill pillKey="tag" pillKeyLabel="tag" pillValue={imageVersion.tag} pillValueLabel={imageVersion.tag} />
-                <Pill
-                  pillKey="repository"
-                  pillKeyLabel="repository"
-                  pillValue={imageVersion.repository}
-                  pillValueLabel={imageVersion.repository}
-                />
-                <Pill
-                  pillKey="version"
-                  pillKeyLabel="version"
-                  pillValue={imageVersion.version}
-                  pillValueLabel={imageVersion.version}
-                />
-              </Stack>
-            </Stack>
-
-            {/* Issues Count Row */}
-            <Stack gap="2" direction="horizontal">
-              <Label text="Number of Issues: " />
-              <Stack direction="horizontal" gap="4" alignment="center">
-                <Stack direction="horizontal" gap="2" alignment="center">
-                  <span>Critical:</span>
-                  <Badge
-                    icon="danger"
-                    text={`${imageVersion.issueCounts?.critical || 0}`}
-                    variant={imageVersion.issueCounts?.critical > 0 ? "danger" : "default"}
+    <MessagesProvider>
+      <Messages />
+      <Panel heading={`Image ${imageVersion.repository} Information`} opened={true} onClose={onClose} size="large">
+        <PanelBody>
+          <Stack gap="6" direction="vertical" className="w-full">
+            <Stack gap="4" direction="vertical">
```

Comment:
You could use simple margin or pading to create space between the items instead of this Stack.